cts/cts-cli.in

```diff
@@ -2289,6 +2289,38 @@ function test_tools() {
     desc="Switch to nonexistent shadow instance (force) (XML)"
     cmd="crm_shadow --switch $shadow --batch --force --output-as=xml"
     test_assert_validate $CRM_EX_NOSUCH 0
+    
+    export CIB_file_invalid_1="$test_home/cli/crm_verify_invalid_bz.xml"
+    export CIB_file_invalid_2="$test_home/cli/crm_verify_invalid_no_stonith.xml"
+    export CIB_file_invalid_3="$test_home/cli/crm_verify_invalid_xml_error.xml"
+
+    desc="Verbosely verify a file-specified invalid configuration, ouputting as xml"
+    cmd="crm_verify --xml-file $CIB_file_invalid_1 --output-as=xml --verbose"
+    test_assert_validate $CRM_EX_CONFIG 1
```

Comment:
The final argument here is whether the CIB should be displayed in output after running the command. It only needs to be 1 when the command changes the CIB -- since crm_verify doesn't, it can be 0 in these cases